services/gitlab/gitlab-contributors.tester.js

```diff
@@ -0,0 +1,52 @@
+import { createServiceTester } from '../tester.js'
+import { isMetric } from '../test-validators.js'
+export const t = await createServiceTester()
+
+t.create('Contributors')
+  .get('/guoxudong.io/shields-test/licenced-test.json')
+  .expectBadge({
+    label: 'contributors',
+    message: isMetric,
+  })
+
+t.create('1 contributor')
+  .get('/guoxudong.io/shields-test/licenced-test.json')
+  .expectBadge({
+    label: 'contributors',
+    message: '1',
```

Comment:
Service code is looking good now :+1: 

In terms of the tests, we don't ideally want to test individual values at the service level. This one will just break if a second person contributes to this repo and it is not really testing anything beyond what the test above using `isMetric`.

With the "5k" test: `metric()` itself has its own test suite https://github.com/badges/shields/blob/8f7164f96bcdb097c310797974541b45d33ed58c/services/text-formatters.spec.js#L40 so we don't really need to re-test the functionality of this in every badge that uses it.

I think we can drop down to just the

- 'Contributors'
- 'Contributors (repo not found)'
- 'Mocking the missing x-total header'

tests and ditch the other 2.

Aside from that, I think this is good to merge. Thanks

Reply:
To prevent misunderstanding, you mean to delete the following two test cases：
- `1 contributor`
-  `Mocking 5k contributors`

Is it right?